build/media_source/mod_accessibility/js/accessibility.es6.js

```diff
@@ -298,8 +342,32 @@
   let scalingEnabled = 0;
   window.fontPercent = 0;
 
-  let validTags = ['p', 'div', 'span', 'i', 'b', 'strong', 'section', 'article', 'h1', 'h2', 'h3', 'h4',
-    'h5', 'h6', 'table', 'tr', 'th', 'td', 'a', 'ul', 'ol', 'li', 'input', 'button'];
+  let validTags = [
```

Comment:
What about custom elements or other general element such as `select`?

Reply:
or em or nav

Reply:
The accessibility module needs more jobs to do. It is in its initial stage.

Reply:
@ahamed07sajeeb please raise this as a separate issue, referencing this PR so as that charlies effort is not lost